x/auth/client/cli/query.go

```diff
@@ -146,8 +146,17 @@ func GetAccountAddressByIDCmd() *cobra.Command {
 				return err
 			}
 
+			accNumUint, err := strconv.ParseUint(args[0], 10, 64)
```

Comment:
```suggestion
			accNum, err := strconv.ParseUint(args[0], 10, 64)
```

Reply:
`accNum` is already being used to `parseInt` at L144

Reply:
I don't understand why we're parsing `args[0]` twice? We should only be calling `strconv.ParseUint(args[0], 10, 64)`. Remove the `accNum, err := strconv.ParseInt(args[0], 10, 64)` line 👍